app/views/employers/employer_attestations/_verify_attestation.html.erb

```diff
@@ -59,7 +59,7 @@
                 </div>
                 <div class="col-sm-9">
                   <div class="form-group rejection_reason hidden">
-                    <%= select_tag "reason_for_rejection", options_for_select(EmployerAttestationDocument::REASON_KINDS), prompt: "Reason for rejection" %>
+                    <%= select_tag "reason_for_rejection", options_for_select(EmployerAttestationDocument::REASON_KINDS), prompt: "Reason for rejection",onchange: "myFunction()" %>
                   </div>
```

Comment:
Use a useful function name that actually describes its purpose. not 'myFunction()'